webapp/src/main/java/org/iqu/webapp/factory/ServiceFactory.java

```diff
@@ -0,0 +1,68 @@
+package org.iqu.webapp.factory;
+
+import orq.iqu.slaveservices.events.EventsService;
+import orq.iqu.slaveservices.events.EventsServiceImpl;
+import orq.iqu.slaveservices.news.NewsService;
+import orq.iqu.slaveservices.news.NewsServiceImpl;
+import orq.iqu.slaveservices.others.OtherService;
+import orq.iqu.slaveservices.others.OtherServiceImpl;
+import orq.iqu.slaveservices.others.RetrieveImageServiceSlave;
+import orq.iqu.slaveservices.others.RetrieveImageServiceImpl;
+import orq.iqu.slaveservices.others.RetrieveLocationService;
+import orq.iqu.slaveservices.others.RetrieveLocationServiceImpl;
+
+public class ServiceFactory {
+
+	private static ServiceFactory instance = null;
+	private static EventsService eventsServiceInstance = null;
+	private static NewsService newsServiceInstance = null;
+	private static OtherService healthCheckServiceInstance = null;
+	private static RetrieveLocationService locationServiceInstance = null;
+	private static RetrieveImageServiceSlave imageServiceInstance = null;
+
+	private ServiceFactory() {
+
+	}
+
+	public static ServiceFactory getInstance() {
```

Comment:
these methods should be thread-safe otherwise you mind end up with multiple instances instead of just one. A simple null check is not enough.